pegasus/sites.v3/code.org/public/css/page/helloworld-styles.scss

```diff
@@ -0,0 +1,76 @@
+@import 'color', 'font', 'typography', 'breakpoints', 'barlow-semi-condensed-font';
+
+// Hero banner
+.hero-banner-basic {
+
+    &.overlay {
+      position: relative;
+      width: 100%;
+      height: 100%;
+  
+      &::before {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        background: #005F6A;
+        z-index: 1;
+      }
+  
+      &::after {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        background-image: url('/images/banners/banner-bg-squiggle-lines.png');
+        background-repeat: repeat;
+        background-size: contain;
+        z-index: 2;
+      }
+    }
+  
+    @media screen and (max-width: $width-sm) {
+      background-size: 50%;
+    }
+  
+    .wrapper {
+      position: relative;
+      z-index: 10;
+    }
+  
+    .text-wrapper {
+      width: 64%;
+  
+      @media screen and (max-width: $width-sm) {
+        width: 100%;
+      }
+    }
+  
+    .media-wrapper {
+      margin-left: 2rem;
+      width: 40%;
+  
+      @media screen and (max-width: $width-sm) {
+        display: none;
+      }
+  
+      img {
+        width: 100%;
+      }
+    }
+  }
+
+  .helloworld_student_projects {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+.action-block--two-col {
+    margin-top: 2rem;
+}
```

Comment:
Nit: there's a class for this too: `.add-margin-top`.

Reply:
Added/deleted!